update.sh

```diff
@@ -131,9 +131,9 @@ for version in "${versions[@]}"; do
               ' "$buildDir/$version/$variant/$distribution/$debug/Dockerfile" > "$buildDir/$version/$variant/$distribution/$debug/Dockerfile.new"
               mv "$buildDir/$version/$variant/$distribution/$debug/Dockerfile.new" "$buildDir/$version/$variant/$distribution/$debug/Dockerfile"
 
-              sed -i "s/#%%INFO_HEADER%%/${infoHeader}/" "$buildDir/$version/$variant/$distribution/$debug/"*
+              gsed -i "s/#%%INFO_HEADER%%/${infoHeader}/" "$buildDir/$version/$variant/$distribution/$debug/"*
```

Comment:
`update.sh` is now obsolete anyway, or is it still used somewhere, no? 😊

Reply:
no, can be deleted.

Reply:
I'll figure out the build issues first, and then we can clean up